### DISCUSSION TITLE
Embed from Metabase

### DISCUSSION THREAD

**Discussion by joeDH00** [Budibase/budibase#12977] [Help]
I created a dashboard in metabase using the same database as the budibase. After that, I enable public sharing and tried to embed the dashboard to budibase embed component and got an error.

![image](https://github.com/Budibase/budibase/assets/159232925/30a6c8fc-1688-4d13-b589-40a92c536103)

Upon inspect, this is the error i gotten

`Refused to frame 'http://110.230.14.155:10403/' because it violates the following Content Security Policy directive: "frame-src 'self' https:".`

Is there a way to fix it. Few things to note:
1. both metabase and budibase are self hosted in my staging server
2. I tried looking up solutions and one solution is to use **https** over **http** but this is in a staging server so i dont think i could do it

**Comment by ConorWebb96** [comment]
The CORS error you are seeing above is simply down to you not hosting your Budibase instance via `https`. Get an SSL cert and secure your platform and you should be able to use iframes going forward.

You should be able to secure it even in a staging server.

**Reply by joeDH00** [reply]
so i need to host both the budibase and metabase via https for me to use the iframes from metabase to budibase?
